docs/en/Blog-Posts/2023-02-08 v7_1_Preview/POST.md

```diff
@@ -0,0 +1,207 @@
+# ABP.IO Platform 7.1 RC Has Been Released
+
+Today, we are happy to release the [ABP Framework](https://abp.io/) and [ABP Commercial](https://commercial.abp.io/) version **7.1 RC** (Release Candidate). This blog post introduces the new features and important changes in this new version.
+
+Try this version and provide feedback for a more stable version of ABP v7.1! Thanks to all of you.
+
+## Get Started with the 7.1 RC
+
+Follow the steps below to try version 7.1.0 RC today:
+
+1) **Upgrade** the ABP CLI to version `7.1.0-rc.1` using a command line terminal:
+
+````bash
+dotnet tool update Volo.Abp.Cli -g --version 7.1.0-rc.1
+````
+
+**or install** it if you haven't before:
+
+````bash
+dotnet tool install Volo.Abp.Cli -g --version 7.1.0-rc.1
+````
+
+2) Create a **new application** with the `--preview` option:
+
+````bash
+abp new BookStore --preview
+````
+
+See the [ABP CLI documentation](https://docs.abp.io/en/abp/latest/CLI) for all the available options.
+
+> You can also use the [Get Started](https://abp.io/get-started) page to generate a CLI command to create a new application.
+
+You can use any IDE that supports .NET 7.x, like [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/).
+
+## Migration Guides
```

Comment:
Change title to "Migrating to 7.1" and write that: "This version doesn't introduce any breaking change. However, Entity Framework developers may need to add a new code-first database migration to their projects since we made some improvements on existing entities of some application modules."